Rocket.Chat/Controllers/Subscriptions/SubscriptionsViewController.swift

```diff
@@ -78,6 +83,12 @@ final class SubscriptionsViewController: BaseViewController {
         tableView.reloadData()
         registerKeyboardHandlers(tableView)
     }
+
+    @IBAction func clearSearch(_ sender: UIButton) {
+        textFieldSearch.text = ""
+        searchBy()
```

Comment:
UITextField delegate's `- textField:shouldChangeCharactersInRange:replacementString:` will  not be called while setting UITextField's text programmatically。